coalib/bearlib/aspects/Redundancy.py

```diff
@@ -97,6 +97,16 @@ class docs:
         Usually, unused imports can simply be removed.
         """
 
+    remove_non_standard_import = Taste[bool](
+        "Remove ALL unused import, include those not from language's "
+        'standard library.',
+        (True, False), default=True)
+
+    expand_wildcard_imports = Taste[bool](
```

Comment:
Note that this feature still hasn't released from upstream so I think I'll just remove this.